src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+:root {
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  box-sizing: border-box;
```

Comment:
Avoid fixing the container size like setting a width of 100% here. The block element like 'header' will naturally expand to the full width of its parent, so setting the width to 100% is unnecessary unless you have a specific reason for doing so.